### ISSUE TITLE
Lab for streamflow statistics

### ISSUE THREAD

**Issue by dblodgett-usgs** [dblodgett-usgs/NWCEd#6]
I'm opening an issue for this to move the conversation to a more traceable place.

from @jnelson7 

```
I am in the process of creating a lab that walks users through the
Streamflow Stats tool.  When you have a moment, I am hoping you can point
me towards some sources I can read to learn how the stream gages have been
organized into the 4 categories: Gages for Evaluating Streamflow, Gages
Active in 2009, Gages 2 Reference Gages, and Gages Sorted by Period of
Record.  I was thinking it would be good to provide the users with some
explanation on this.
```

This dataset: 

http://water.usgs.gov/GIS/metadata/usgswrd/XML/gagesII_Sept2011.xml

https://www.researchgate.net/publication/273203633_GAGES_II_Geospatial_Attributes_of_Gages_for_Evaluating_Streamflow_summary_report

is the source of the attributes that created those map filters.

**Comment by jnelson7** [comment]
Thank you so much.

On Thu, May 5, 2016 at 1:33 PM, David Blodgett notifications@github.com
wrote:

> I'm opening an issue for this to move the conversation to a more traceable
> place.
> 
> from @jnelson7 https://github.com/jnelson7
> 
> I am in the process of creating a lab that walks users through the
> Streamflow Stats tool.  When you have a moment, I am hoping you can point
> me towards some sources I can read to learn how the stream gages have been
> organized into the 4 categories: Gages for Evaluating Streamflow, Gages
> Active in 2009, Gages 2 Reference Gages, and Gages Sorted by Period of
> Record.  I was thinking it would be good to provide the users with some
> explanation on this.
> 
> This dataset:
> 
> http://water.usgs.gov/GIS/metadata/usgswrd/XML/gagesII_Sept2011.xml
> 
> https://www.researchgate.net/publication/273203633_GAGES_II_Geospatial_Attributes_of_Gages_for_Evaluating_Streamflow_summary_report
> 
> is the source of the attributes that created those map filters.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/dblodgett-usgs/NWCEd/issues/6

## 

Jake Nelson

**Comment by dblodgett-usgs** [comment]
Fixed by #7 👍
